playpen/ansible/roles/dev/files/bashrc

```diff
@@ -82,6 +82,16 @@ pclean() {
     done
     popd
 
+    # Scrub out the Squid cache
+    if [ -d /var/spool/squid ]; then
+        sudo systemctl stop squid
+        sudo rm -rf /var/spool/squid
+        sudo mkdir --context=system_u:object_r:squid_cache_t:s0 --mode=750 /var/spool/squid
```

Comment:
It is indeed not readable by regular users which is why I resorted to removing the directory, but this is better.